### PR TITLE
SJSU-1066, Disclaimer Page

### PR DESCRIPTION
**Pull request by yuta05**
Added the content into the disclaimer page.

### CODE REVIEW

File: routes/api/bookin.js [hunk 1]
```diff
@@ -95,7 +95,26 @@ router.get("/history", passport.authenticate("jwt", { session: false }), (req, r
                             });
 
                             // Check if we've finished packing ALL the history objects
-                            if (historyPack.length === bookings.length) return res.status(200).send(historyPack);
+                            if (historyPack.length === bookings.length) {
```
Comment (SandroSal) [inline routes/api/bookin.js]: what is this?

File: front_end/src/components/about_page/AboutUs.js [hunk 2]
```diff
@@ -128,7 +128,7 @@ export default (class aboutUs extends Component {
               </Grid>
               <Grid item className="teamMemberContainer">
                 <img className="aboutPage-images" src={Yuta} alt="" />
-                <br /> <div id="AboutPageNames">Yuta</div>{" "}
+                <br /> <div id="AboutPageNames">Yuta Sugiura</div>{" "}
```
Comment (SandroSal) [inline front_end/src/components/about_page/AboutUs.js]: sorry that your last name wasn't there... don't know why that is...

File: front_end/src/components/disclaimer_page/disclaimer.css [hunk 3]
```diff
@@ -0,0 +1,24 @@
+#DisclaimerContainer {
```
Comment (SandroSal) [inline front_end/src/components/disclaimer_page/disclaimer.css]: is this used?

Comment (yuta05) [reply front_end/src/components/disclaimer_page/disclaimer.css]: deleted

File: routes/api/bookin.js (hunk 1 above)
Comment (yuta05) [reply routes/api/bookin.js]: I don't know why this happened... I fixed it back

**Pull request closed without merging**